Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+import UIKit
+import MapboxMaps
+import Turf
+
+private enum SymbolTailPosition: Int {
+    case left
+    case right
+    case center
+}
+
+private struct DebugFeature {
```

Comment:
Can we choose a different name here? What's Debug about this feature?

Reply:
Another name is fun. It was originally named debug since it was for debugging the visualization of the annotations and "Feature" was a reserved word. Maybe "AnchoredFeature" or similar is better now.

Reply:
See https://github.com/mapbox/mapbox-maps-ios/pull/548#discussion_r673294649. I think avoiding the term feature in this name would be advantageous.